DuckDuckGo/LargeOmniBarState.swift

```diff
@@ -54,6 +48,7 @@ struct LargeOmniBarState {
         var onBrowsingStoppedState: OmniBarState { return self }
         var onEnterPadState: OmniBarState { return self }
         var onEnterPhoneState: OmniBarState { return SmallOmniBarState.HomeEmptyEditingState() }
+        var onReloadState: OmniBarState { return BrowsingNonEditingState() }
```

Comment:
return is not needed, even though it matches the convention from lines above I believe we shouldn't use it for 1-liners - actually it would be good to have a rule for that, wdyt?

Reply:
yep, but since the other ones have return I thought it was better to follow the pattern. But I agree we should have a rule for that. I don't think it's part of this PR though.